cpp/src/structure/induced_subgraph_impl.cuh

```diff
@@ -206,7 +208,7 @@ extract_induced_subgraphs(
       thrust::make_counting_iterator(num_aggregate_subgraph_vertices),
       [subgraph_offsets,
        subgraph_vertices,
-       num_subgraphs,
+       num_subgraphs = subgraph_offsets.size() - 1,
```

Comment:
It seems like `num_subgraphs` is not used?

Reply:
Removed